ui/test/dashboards/reducers/cellEditorOverlay.test.js

```diff
@@ -30,11 +30,20 @@ const defaultCellAxes = {
   },
 }
 
+const defaultCellTableOptions = {
```

Comment:
you can just refer to `DEFAULT_TABLE_OPTIONS` from the constants file above instead of redefining a constant